src/main/java/picard/illumina/ExtractIlluminaBarcodes.java

```diff
@@ -174,6 +183,9 @@
             "the number available on the machine less NUM_PROCESSORS.")
     public int NUM_PROCESSORS = 1;
 
+    @Option(doc = "Use the new converter", optional = true)
```

Comment:
Yeah, a big 👎  if you need to intelligently set this based on sequencer type.  These demux tools already hard enough to run.